en/docs/tutorials/construct-statements.md

```diff
@@ -0,0 +1,119 @@
+
+5. Click the **var** type descriptor on the statement and click **string** in the **Suggestions** tab.
+
+    !!! info
+        The **Suggestions** tab provides contextual suggestions based on the selected item. The **Suggestions** tab currently lists variable type suggestions because you selected the type descriptor.    
+
+    ![Change type descriptor](../assets/img/statement-editor/change-type-descriptor.gif){.cInlineImage-half}
+
+    Here, you are specifying that the variable value is a string.
+
+6. To edit the variable name, double-click **`variable`** and type `maxValue` in the text box.
+
+    !!! info
+        Another way to make a part of a statement editable is by clicking on the required statement component and pressing  **Enter**/**Return**.
+
+    ![Edit variable name](../assets/img/statement-editor/edit-variable-name.gif){.cInlineImage-half}
+
+7. Double click **`<add-expression>`** and type `"Max : "` in the text box.
+
+    !!! info
+        To navigate from one item on the statement to another, you can also press **Tab** / **Shift** + **Tab**.
+
+    ![Edit variable expression](../assets/img/statement-editor/edit-variable-expression.gif){.cInlineImage-half}
+
+    Here, you are entering static text as a part of the variable value.
+
+8. Click **Expressions**, and under the **Arithmetic** section, click the **Es + Ex** expression.
+
+    !!! info
+        The **Expressions** tab lists all the pre-defined templates that you can use to define complex expressions.
+
+    ![Select expression template](../assets/img/statement-editor/select-expression-template.gif){.cInlineImage-half}
+
+    In the `Es + Ex` expression template, `Es` refers to the existing expression you clicked and selected. `+ Ex` combines another expression with `Es`.
+
+9. Click **Libraries** and search for `max`. In the filtered list, click  **lang.int:max**.
+
+    !!! info
+        The **Libraries** tab lists all the Ballerina Standard and Language libraries.
+
+    ![Select library](../assets/img/statement-editor/select-library.gif){.cInlineImage-half}
+
+    `lang.int:max` is a Ballerina standard library for deriving the maximum value.
+
+     You will see some diagnostics generated for the library you selected. Move on to the next step to resolve them. 
+
+10. Click the **int0:max(n,ns)** function. 
+
+     When you select a function or a method call, the **Parameters** tab opens by default. This tab displays information about the selected function or method call, including details about parameters, descriptions, and return type descriptions. In this tab, you can easily add parameters to the expression by selecting the check boxes for the relevant parameters.
+
+     ![Select function](../assets/img/statement-editor/select-function.gif){.cInlineImage-half}
+
+     !!! note
+         When a parameter requires a function or a method call to work, the Statement Editor selects its check box for that parameter by default and does not allow you to clear it.
```

Comment:
is this sentence giving the right idea?
Shouldn't it be 
"When a function or a method call requires a particular parameter to work, the Statement Editor selects the appropriate parameter by default and does not allow you to clear it."

Reply:
Corrected as pointed out.